### PR TITLE
Billw rec

### LINKED ISSUE #12: BillW checklist
- [x] Create a branch
-  [x] Add a file
- [ ] Make a PR
- [ ] Merge

### PR DESCRIPTION
**Pull request by Bill-Watts**
Closes #12

### CODE REVIEW

**Comment by Bill-Watts** [comment]
Mark please review.

**Comment by Bill-Watts** [comment]
Adding recommendations.

Closes #18

File: billw-rec.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Farmington, MN
+
+### Places to eat
+- My dinner table
```
Comment (mark-boatman) [inline billw-rec.md:4]: What is for dinner?

File: billw-rec.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+- My desk
```
Comment (mark-boatman) [inline billw-rec.md]: Easier to get work done when you eat at your desk.  Good call! :taco:

**Review by mark-boatman — COMMENTED** [review]
Nice work.  Good summary.

**Review by mark-boatman — COMMENTED** [review]
@Bill-Watts looks good.

**Review by mark-boatman — APPROVED** [review]
Looks awesome. Get these changes deployed asap.

**Comment by Bill-Watts** [comment]
Squashing this merge pull.

**Pull request merged**